aiohttp/multipart.py

```diff
@@ -854,6 +856,8 @@ def _apply_content_transfer_encoding(self, stream):
         elif encoding == 'quoted-printable':
             for chunk in stream:
                 yield binascii.b2a_qp(chunk)
+        elif encoding == 'binary':
+            yield from stream
```

Comment:
@AleksandrIakhnev Sorry, but I failed to reproduce your issue: script you provide works for me returning 405 response from Google. I also add [test](https://github.com/KeepSafe/aiohttp/pull/1169/files#diff-a5c957a9c2be1397119d89b7aea0a7efR182) about to make sure that it works right and expected.

Are you sure that nothing is missed there in your example?